### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by khushijohri2001**
Please review my code for my Component Library's Navigation Component.

The page includes:

Header with Girlifycoder Logo, Heading, and Github link
The navigation bar on the left
Sidebar on the right
Details about the Component and snippet

https://user-images.githubusercontent.com/65448358/160251148-d43a73b1-d4c1-43c3-b8e6-5cc8f1eb3547.mp4

### CODE REVIEW

File: navigation/styles2-navigation.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+.navigation-sub {
+  margin: 1rem 0rem;
+  color: white;
+  font-size: 1.2rem;
+}
+.navigation-snippet {
+  margin: 1rem 0rem;
+}
+.standard-navigation {
+  color: yellow;
```
Comment (pnchinmay) [inline navigation/styles2-navigation.css]: Change these to color variable with hex or rgb code

File: navigation/styles2-navigation.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  color: var(--primary-color);
+}
+
+.drawer-nav-navigation {
+  border: 1px black solid;
+  display: inline-block;
+  padding-right: 2rem;
+  border-radius: 1rem;
+  padding-bottom: 1rem;
+  background-color: #a517ba;
```
Comment (pnchinmay) [inline navigation/styles2-navigation.css:88]: Color variables are recommended

**Review by pnchinmay — APPROVED** [review]
Left a few comments
Everything else looks good :100:

File: navigation/styles2-navigation.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  margin: 3rem;
+}
+
+.navigation-main-heading {
+  color: yellow;
+}
+.navigation-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
+  height: 200px;
```
Comment (i-am-a-codegirl) [inline navigation/styles2-navigation.css:12]: use rems instead

**Review by i-am-a-codegirl — APPROVED** [review]
Looks good to merge

File: navigation/styles2-navigation.css (hunk 3 above)
Comment (khushijohri2001) [reply navigation/styles2-navigation.css:12]: will do it next time..thanks for suggesting

File: navigation/styles2-navigation.css (hunk 2 above)
Comment (khushijohri2001) [reply navigation/styles2-navigation.css:88]: done

**Pull request merged**